### PR TITLE
Plots of continuous distributions

### PR DESCRIPTION
**Pull request by aasensio**
This is a simple PR to add plots of continuous distributions in the documentation. This is something that I miss sometimes and I end up going to the Wikipedia for that. I think it's good to have it in the documentation.

### CODE REVIEW

**Comment by junpenglao** [comment]
This is a nice idea - but a rst page in the doc might be better than in the docstring?

**Comment by twiecki** [comment]
That's very cool. I think it makes sense to have this in the docs.

File: pymc3/distributions/continuous.py [hunk 1]
```diff
@@ -117,6 +117,25 @@ class Uniform(Continuous):
...
 
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        x = np.linspace(-3.0, 3.0, 1000)
+        a, b = 0.0, 2.0
+        y = np.zeros(1000)
+        y[(x<b) & (x>a)] = 1.0/(b-a)
+        plt.plot(x, y, label='lower=1, upper=2')
```
Comment (twiecki) [inline pymc3/distributions/continuous.py]: if we moved to `fig, ax = plt.subplots(); ax.plot()...` we could save a few lines with `ax.set(ylim=..., xlabel=...)`

Comment (aasensio) [reply pymc3/distributions/continuous.py]: Done. Makes sense.

**Comment by twiecki** [comment]
do you have it run locally and if so can you post a screen-shot of what the rendered result looks like?

**Comment by aasensio** [comment]
Here you have a couple of screenshots. I guess the results depends on the version of matplotlib used for the docs, that I guess they are done automatically with Travis, right?

screen shot 2017-08-01 at 21 07 10
screen shot 2017-08-01 at 21 07 24

**Comment by twiecki** [comment]
I love it!

**Comment by AustinRochford** [comment]
This is awesome!

**Comment by fonnesbeck** [comment]
This is better than most textbooks. Thanks for the contribution.

**Pull request merged**